client/components/AddGeoLookupInput/index.jsx

```diff
@@ -18,15 +18,17 @@ class GeoLookupInput extends React.Component {
         return (
             <Geolookup
                 placeholder='Address, City'
-                inputClassName="line-input"
-                buttonClassName='btn btn-default geolookup__button'
+                inputClassName={(this.props.readOnly ? 'disabledInput ' : '') + 'line-input'}
```

Comment:
It's totally fine. I just want to let you know that I introduced a `classNames` module that help to define classes like that.
https://github.com/JedWatson/classnames
Use it only if you like :)